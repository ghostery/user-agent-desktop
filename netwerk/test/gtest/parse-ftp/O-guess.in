220 server IBM TCP/IP for OS/2 - FTP Server ver 23:04:36 on Jan 15 1997

This listing is just a guess because I couldn't find an FTP server that 
LISTs like this. This guess is based on comments in Perl mirror project.
I'm particularly unsure of the contents of cols 19-34.

          1         2         3         4         5         6
0123456789012345678901234567890123456789012345678901234567890123456789
----- size -------| ?????????????? MM-DD-YY|  HH:MM| nnnnnnnnn....

                 0  DIR            04-11-95   16:26  .
                 0  DIR            04-11-95   16:26  ..
                 0  DIR            04-11-95   16:26  ADDRESS
               612  A              07-28-95   16:45  air_tra1.bag
               195  A              08-09-95   10:23  Alfa1.bag
                 0  DIR            04-11-95   16:26  ATTACH
               372  A              08-09-95   10:26  Aussie_1.bag
            310992                 06-28-94   09:56  INSTALL.EXE
             12345  H              10-19-00   15:10  ADOE ADIR MOZILLA.FAKEOUT
