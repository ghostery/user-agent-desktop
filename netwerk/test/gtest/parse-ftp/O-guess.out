<!-- This Source Code Form is subject to the terms of the Mozilla Public
   - License, v. 2.0. If a copy of the MPL was not distributed with this
   - file, You can obtain one at http://mozilla.org/MPL/2.0/. -->

04-11-1995  16:26:00       <DIR>          .
04-11-1995  16:26:00       <DIR>          ..
04-11-1995  16:26:00       <DIR>          ADDRESS
07-28-1995  16:45:00                  612 air_tra1.bag
08-09-1995  10:23:00                  195 Alfa1.bag
04-11-1995  16:26:00       <DIR>          ATTACH
08-09-1995  10:26:00                  372 Aussie_1.bag
06-28-1994  09:56:00               310992 INSTALL.EXE
10-19-2000  15:10:00                12345 ADOE ADIR MOZILLA.FAKEOUT
